src/assets/javascript/components/buttom.js

```diff
@@ -0,0 +1,6 @@
+function buttom(textContent) {
```

Comment:
hey the correct name is with `n`
button

Reply:
true 🫡

Reply:
Yeah please where is using buttom change to button, please ;)
It was my fault :D